cfn-resources/test/e2e/utility/publish_cfn_to_registry.sh

```diff
@@ -37,8 +37,9 @@ mv "${tmp_rpdk_file}" "${rpdk_file}"
 
 
 echo "Creating a new resource schema"
-schema_file_name="${resource_directory//-}"
-echo "schema name file........ : ${schema_file_name}"
+# shellcheck disable=SC2001
+schema_file_name=$(echo "$resource_directory" | sed 's/-//g')
```

Comment:
any reason to disable instead of fixing 
https://www.shellcheck.net/wiki/SC2001

Reply:
The shellcheck was suggesting to use the same pattern that was causing the issue when this script runs in Github actions. Hence, disabled. Please let me know if there's a better way to handle the replacement at line 41.
<img width="1122" alt="Screenshot 2023-05-25 at 10 01 47" src="https://github.com/mongodb/mongodbatlas-cloudformation-resources/assets/122359335/89bb3696-3489-47ca-acc7-e65553040e7d">

Reply:
what's the value of `resource_directory` and what's the value you expect?

Reply:
`resource_directory` can be `third-party-integration`, so we just want to remove the `-` to make it `thirdpartyintegration`

Reply:
```bash
resource_directory="third-party-integration" ; echo "${resource_directory//-/}"
```
works for me, what error were you seeing with the original replacement?

Reply:
Was getting `(https://github.com/mongodb/mongodbatlas-cloudformation-resources/actions/runs/5068142046/jobs/9100062790?pr=570#step:5:43) ../utility/publish_cfn_to_registry.sh: 40: Bad substitution` as mentioned in [INTMDB-839](https://jira.mongodb.org/browse/INTMDB-839)
Updated to `${resource_directory//-/}` and it works now. Thanks!